lumicks/pylake/simulation/diffusion.py

```diff
@@ -34,9 +34,11 @@ def simulate_wiener(sigma, num_steps, time_step):
 def _get_blank_kymo(*, line_time_seconds):
     """Generate kymo with particular properties"""
 
-    return _kymo_from_array(
+    kymo = _kymo_from_array(
         np.zeros((0, 0)), "r", line_time_seconds=line_time_seconds, pixel_size_um=1.0
     )
+    kymo._motion_blur_constant = 0
```

Comment:
we don't have to do it now, but I guess we should add this as a parameter for `_kymo_from_array`

Reply:
The reason I would hold off on that for now is that I'd rather compute it from the exposure rather than expose the blur constant directly. I would pick this up if/when we start supporting diffusive analysis on camera images.